cue.yaml

```
@@ -0,0 +1,13 @@
+module: github.com/banzaicloud/istio-operator
```

Again, not sure if matters here, but maybe:
```suggestion
module: github.com/banzaicloud/istio-operator/v2
```